server/routes/users.js

```diff
@@ -116,10 +116,41 @@ export default (storage, scriptManager) => {
 
             return {
               user,
-              memberships: [ ]
+              memberships: []
             };
           });
       })
+      .then(data => {
+        if (!data.user.identities) {
+          data.connection = {};
+          return data;
+        }
+
+        const identities = data.user.identities.filter(identity => identity.provider === 'auth0');
+        const name = identities[0] && identities[0].connection;
+
+        if (!name) {
+          data.connection = {};
+          return data;
+        }
+
+        return req.auth0.connections.getAll({ name, fields: 'id' })
+          .then((connection) => {
```

Comment:
It bothers me that we have to get all connection ID's before we can find the ID for this particular connection, but I'm not sure there is a better way.  You might want to just cache the connections in the global object.  That would limit the number of requests, but this is still quite a bit better than it was before.

Reply:
One option would be to move this to a separate call, then the FE could find the ID in its map, which it already has in state, and then just call back here to get the list of enabled clients for that connection.  Also, that call to get list of ID's should be paged, otherwise it could be huge for certain customers.

Reply:
I don't like the idea of loading in PasswordResetDialog, so i would prefer getting all data in one place.
Added global.connections (contains list of `{id, name}`).

I'll do separate endpoint (and request from FE), if you think it would be better.